make_tag.py

```diff
@@ -127,20 +129,30 @@ def make_tag(remote, bump_major, bump_minor, need_verification):
             print('Skipped creating the tag')
             return
 
-    tag_success = call(['git', 'tag', '-a', '-m', tag_message, tag_name])
-    if tag_success != 0:
-        sys.exit()
-    call(['git', 'push', remote, tag_name])
+    if github_token:
+        headers = {
+            'Authorization': f'token {github_token}',
+            'Content-Type': 'application/x-www-form-urlencoded',
+        }
+        response = requests.post('https://api.github.com/repos/linkedin/venice/git/refs', headers=headers, json={"ref": "refs/tags/" + tag_name, "sha" : "$commit"})
```

Comment:
`$commit` needs to be an actual commit reference

two nits:
1. Let's use `''` where possible (to follow the pattern of the rest of the file)
2. we can use `fstring` for `tag_name` now: `f'refs/tags/{tag_name}'`